packages/documentation/src/pages/forms/form-tutorial-basic.mdx

```diff
@@ -34,7 +35,13 @@ You can answer the questions with the [documentation](https://github.com/departm
 - What's the benefit description for this form? `new form benefits`
 - Which form template would you like to start with? `Choose 'BLANK: A form without any fields'`
 
-After you run the generator, run `yarn watch` (or restart it if you already had it running) and navigate to http://localhost:3001/new-form. You should see something like this:
+After you run the generator, run
+
+```bash
+yarn watch
+```
+
+If you already had it running, restart the watch commmand. After the watch task successfully starts up, navigate to http://localhost:3001/new-form (or the path you entered when asked by the generator). You should see something like this:
```

Comment:
Maybe add this as a note after the start step 

_Restart the `watch` task after running the yeoman generator_

Maybe make the navigation a new step:
- Navigate to http://localhost:3001/new-form